lib/internal/http2/core.js

```diff
@@ -1202,8 +1219,8 @@ function createClientSession(options, socket) {
   socket.on('drain', socketOnDrain);
 
   session.on('error', clientSessionOnError);
-  session.on('stream', sessionOnStream);
-  session.on('selectPadding', sessionOnSelectPadding);
+  // session.on('stream', sessionOnStream);
+  // session.on('selectPadding', sessionOnSelectPadding);
```

Comment:
For handling these, client specific handlers can be used, similar to how a few other client specific events are handled. I'd have to take a look more in depth to see what the changes would need to be. Would you mind adding a `TODO` comment on these to provide client specific implementations?